client/app/dedicated/server/dedicated-server.controller.js

```diff
@@ -530,5 +541,22 @@ angular.module('App').controller('ServerCtrl', (
   }
 
   load();
+
+  // Upgrade to Advance
+  $scope.isEligibleForUpgrade = () => isEligibleForUpgrade();
+  $scope.trackUpgradeToAdvance = () => trackUpgradeToAdvance();
+  $scope.URLS = URLS;
+
+  function trackUpgradeToAdvance() {
+    atInternet.trackClick({
```

Comment:
`track-click` directive cannot be used on `a` directly ?

Reply:
Yes, it could. Updated